modules/core/assets/ships/imperialCapital/imperialCapital.json

```diff
@@ -0,0 +1,443 @@
+{
+    "size": 3.5,
+    "maxLife": 800,
+    "type": "big",
+    "price": 20000,
+    "e1Pos": "0.08 0.37",
+    "e2Pos": "0.08 0.63",
+    "displayName": "Imperial Capital Ship",
```

Comment:
Usually in sci-fi they are referred to as "capital ships" so I just named it that. Also, the name doesn't really matter right now as the capitals cannot be bought.